pkg/trtl/config/config.go

```diff
@@ -61,6 +61,15 @@ func (c Config) IsZero() bool {
 	return !c.processed
 }
 
+// Mark a manually constructed as processed as long as it is validated.
+func (c Config) Mark() (Config, error) {
+	if err := c.Validate(); err != nil {
+		return c, err
+	}
+	c.processed = true
+	return c, nil
+}
```

Comment:
This code is in `pkg/gds/config` so that tests can be manually created without loading the values from the environment; this is useful for `Trtl` too, so I've added it to `pkg/trtl/config`.